src/components/biggive-main-menu/biggive-main-menu.tsx

```diff
@@ -138,26 +93,64 @@ export class BiggiveMainMenu {
         subSubMenuElement.classList.toggle('display-sub-menu');
       };
     });
+  }
 
-    if (!this.usePresetMenuContent) {
-      this.appendPrimaryNavigationLinks();
-      this.appendNavSecondaryLinks();
-    }
+  private getSecondaryNavLinks() {
+    return (
+      <ul>
+        {this.isLoggedIn && (
+          <li>
+            <a href={makeURL('Donate', this.donateUrlPrefix, 'my-account')}>My Account</a>
+          </li>
+        )}
+        <li>
+          <a href={makeURL('Experience', this.experienceUrlPrefix, 's/contact-us')}>Contact Us</a>
+        </li>
+        <li>
+          <a href={makeURL('Experience', this.experienceUrlPrefix, 'charities/s/login')}>Charity Login</a>
+        </li>
+      </ul>
+    );
   }
 
   render() {
-    const slotBasedMenu = () => (
+    /** ten minutes in advance to account for a *very* slow browser following the link and/or clock skew.
+     * There's not really any harm in doing this early unless its soo early that people start sharing the noredirect link more than we'd like.
+     */
+    const CCOpenDate = new Date('2023-11-27T23:50:00+00:00');
+
+    /**
+     * Ten minutes late. Again doing this late doesn't hurt.
+     */
+    const CCCloseDate = new Date('2023-12-05T12:10:00+00:00');
+
+    const now = new Date();
+
+    const ccIsOpenNowIsh = now > CCOpenDate && now < CCCloseDate;
+
+    const homePageLink = ccIsOpenNowIsh ? '/?noredirect' : '/';
```

Comment:
@bdsl What do you think about `homeIsRedirecting` becoming an input property? I think there are a couple of benefits:

1. We can build this once and then avoid needing new component releases to manage new campaigns.
2. We can control the actual redirect behaviour with the same code that controls the redirect-confusion-mitigation behaviour, which means less chance of mistakes or inconsistency within Donate.
3. Looking ahead, it could take us a step closer to applying the redirect policy systematically based on data, instead of changing the code repeatedly.

The obvious drawback is repeating some JS in WordPress, but I don't think it's that important that this is done consistently in WordPress to be honest. Mostly during CC it's accessed either by people who came from the Donate CC pages, or those landing on the CC info page which already has a prominent CTA button pointing to CC23 landing.

Reply:
I'm hoping we can avoid doing a redirects beyond CC23 - I think we agreed it's not an optimal solution but it's the best we can do for now, which is why I made separate tickets DON-925 for CC23 and DON-926 for the future.

So I think I'd rather keep this focused just on CC23, and only make the input property if we think we're likely to want to change these dates easily. In WordPress I guess we'd code it in PHP rather than WordPress, but for now it feels simpler to have it just code in one place.

Reply:
Cool, let's stick with this for now